net/seafile-server/Makefile

```diff
@@ -33,7 +36,11 @@ define Package/seafile-server
     DEPENDS:=+libarchive +libopenssl +glib2 +libsearpc +seafile-ccnet +seafile-seahub +sqlite3-cli +python-mysql +python-urllib3 \
 		+jansson +libevent2 +libevent2-openssl +zlib +libzdb +libsqlite3 +libmysqlclient \
 		+libpthread +libuuid +bash +procps-ng +procps-ng-pkill +SEAFILE_FUSE_SUPPORT:libfuse $(ICONV_DEPENDS)
-    EXTRA_DEPENDS:=seafile-seahub (=6.3.4-1)
+    ifeq ($(SEAHUB_CUR_VERSION),$(PKG_VERSION))
+        EXTRA_DEPENDS:=seafile-seahub (=$(SEAHUB_CUR_VERSION)-$(SEAHUB_CUR_RELEASE))
+    else
+        EXTRA_DEPENDS:=seafile-seahub (=$(PKG_VERSION)-$(PKG_RELEASE))
```

Comment:
> The logic I used is: try to get current PKG_VERSION and PKG_RELEASE from seafile-seahub Makefile. Use them if the versions match. If not, use the version & release of seafile-seahub. Historically, the versions have always matched.

If I am reading it right, you want to always use version of seafile-seahub no matter what ;)

Reply:
Yes.  What I really wanted was to both match the same version, regardless of release number.  I've tried:
```
EXTRA_DEPENDS:=seafile-seahub (=$(PKG_VERSION))
```
```
EXTRA_DEPENDS:=seafile-seahub (=$(PKG_VERSION)-*)
```
``` 
EXTRA_DEPENDS:=seafile-seahub (>=$(PKG_VERSION)-1 <=$(PKG_VERSION)-999999)
```
and
```
EXTRA_DEPENDS:=seafile-seahub (>=$(PKG_VERSION)-1) seafile-seahub ( <=$(PKG_VERSION)-999999)
```
I'll drop this.  Now that I'm giving this extra thought, it won't work because when *seahub increases release, *server's needs to be increased as well, and we're not doing it.  I'll drop it, and add a comment in *seahub's Makefile that this needs to match.  It's the best.